great-idea/css/index.css

```diff
@@ -201,4 +201,164 @@ button:before {
 button:hover:before {
 	width: 100%;
 	color: red;
+}
+
+/* .categories {
+    display: flex;
+    flex-direction: column;
+	flex-wrap: wrap;
+	border: 1px solid red;
+} */
+
+
+.serv {
+	margin-top: 20px;
+	margin-bottom: 20px;
+	font-size: 30px;
+}
+
+.logo {
+	margin-left: 20px;
+}
+
+.Digital-Design {
+	display: flex;
+    flex-direction: column;
+	flex-wrap: wrap;
+	border: 1px solid black;
+	margin-right: 20px;
+}
+
+.UX { 
+	display: flex;
+    flex-direction: column;
+	flex-wrap: wrap;
+	border: 1px solid black;
+	background-color: #f4f3f4;
+}
+
+.Digital-Marketing {
+	display: flex;
+    flex-direction: column;
+	flex-wrap: wrap;
+	border: 1px solid black;
+	background-color: #f4f3f4;
+	margin-right: 20px;
+
+}
+
+.WebDev { 
```

Comment:
Instead of wrapping your sections with repeating CSS you might consider refactoring to make your code more DRY. For example having one class called "top paragraphs" for each of the top 2 paragraphs and having the same CSS for that class.